docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -21,79 +21,115 @@ To complete this guide, you need:
-Version {graalvm-version} is required.
-* The `GRAALVM_HOME` environment variable configured appropriately
-* The `native-image` tool must be installed; this can be done by running `./gu install native-image` from your GraalVM
-`bin` directory
-* Apache Maven 3.5.3+
-* A working C developer environment (see the note below for details)
-* A running Docker
+* A xref:configuring-c-development[working C development environment]
+* GraalVM version {graalvm-version} installed and  xref:configuring-graalvm[configured appropriately]
+* A running Docker daemon
 * The code of the application developed in the link:getting-started-guide.html[Getting Started Guide].
 
+.Supporting native compilation in C
+[[configuring-c-development]]
 [NOTE]
 ====
-Once you have downloaded GraalVM, expand the archive and set the `GRAALVM_HOME` variable to this location:
+What does having a working C developer environment mean?
 
-`export GRAALVM_HOME=$HOME/Development/graalvm/`
+* On Linux, you will need GCC, and the glibc and zlib headers. Examples for common distributions:
++
+[source,shell]
+----
+# YUM
+sudo dnf install gcc glibc-devel zlib-devel
+# Debian-based distributions:
+sudo apt-get install build-essential libz-dev
+----
+* XCode provides the required dependencies on macOS:
++
+[source,shell]
+----
+xcode-select --install
+----
+====
 
-On MacOS, point the variable to the `Home` sub-directory:
+[[configuring-graalvm]]
+=== Configuring GraalVM
 
-`export GRAALVM_HOME=$HOME/Development/graalvm/Contents/Home/`
+[TIP]
 ====
-
-[NOTE]
+If you cannot install GraalVM, you can use a multi-stage Docker build to run Maven inside a Docker container that embeds GraalVM. There is an explanation of how to do this at the end of this guide.
 ====
-What does having a working C developer environment mean?
 
- * On Linux, you will need GCC, the glibc and zlib headers (on common distributions: `sudo dnf install gcc glibc-devel zlib-devel` or `sudo apt-get install build-essential libz-dev`).
- * On macOS, execute `xcode-select --install`.
-====
+Version {graalvm-version} is required. Using the community edition is enough.
 
-[NOTE]
-====
-Some previous releases of GraalVM included the `native-image` tool by default.  This is no longer the
-case; it now has to be installed as a second step after GraalVM itself is installed.  Use `gu install native-image`
-to do so.
-====
+1. Install GraalVM if you haven't already. You have a few options for this:
+** Use platform-specific install tools like https://github.com/graalvm/homebrew-tap[homebrew] or https://sdkman.io/jdks#Oracle[sdkman].
+** Download the appropriate Community Edition archive from <https://github.com/oracle/graal/releases>, and unpack it like you would any other JDK.
+** Follow https://www.graalvm.org/docs/getting-started/#install-graalvm[instructions] to install the Enterprise Edition
+2. Configure the runtime environment
+** Set `GRAALVM_HOME` environment variable to the GraalVM installation directory, for example:
```

Comment:
Why `**`? We have only one item AFAICS.

Reply:
You're right. That one doesn't need to be a list.